compiler/src/iree/compiler/Dialect/Stream/IR/StreamOps.cpp

```diff
@@ -27,6 +28,10 @@
 #include "mlir/Support/LogicalResult.h"
 #include "mlir/Transforms/RegionUtils.h"
 
+namespace mlir::iree_compiler {
+using IREE::Encoding::getEncodingAttr;
+}
+
```

Comment:
We do not need this. They are declared in the `EncodingTypes.h` and you already include it.